HW_1/templates/base.html

```diff
@@ -0,0 +1,36 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Children's toy store</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+<header>
+    <a class = "menu" href = "index.html">Home</a>
+    <a class = "menu" href = "catalog.html">Сatalog</a>
```

Comment:
Тут должны быть маршруты,  не  пути до `html` файлов
```suggestion
    <a class = "menu" href = "/">Home</a>
    <a class = "menu" href = "/catalog">Сatalog</a>
```

Reply:
После этого изображения из `static` будут подгружаться